script/1_0_2current_en_script/Arcueid/Day 6/06_00_ARC06_1B.txt

```diff
@@ -1,40 +1,40 @@
 <Page2>
-I confirmed with my hands that the cuts I received from Vlov's ice fragments on areas such as my cheeks and shoulders had completely recovered.
-The unnatural lack of even a scar makes it clear that it was the work of Arcueid. She did say that she would treat me after all, that woman.
+I felt around with my hands to confirm, but it seems that the cuts I received on my cheeks and shoulders from Vlov's ice shards have healed completely.//手で確認したが、ヴローヴの氷の破片で受けた頬や肩の切り傷は完治していた。
+The unnatural lack of any scars makes it clear that this is the result of Arcueid's handiwork. She did say that she would treat me after all, that woman.//傷跡さえない不自然さから、アルクェイドの仕業なのは明白だ。手当てをする、とか言ってたし、あいつ。
 <Page3>
-"Shiki-sama. I am sorry to be so intrusive, but please use this."
-"?"
+"Shiki-sama. My apologies for being so forward, but please make use of this."//志貴さま。さしでがましいこととは存じますが、こちらをお使いください
+"Hm?"
 <Page4>
 Hisui timidly presents a long and narrow rod.
-It's a cane... no, a walker.
+It's a cane... no, a crutch.//"歩行器" does seem to mean a Zimmer frame, but those aren't a long thin rod? Translated as crutch since that makes more sense.
 <Page5>
-"So you did have something like this, huh."
-"Yes. I had remembered seeing it in the equipment room previously. So I searched for it with that sliver of hope in mind."
+"Oh, this is perfect."//Assuming this is よく合った? よくあったね、こんなの
+"Yes. I had remembered seeing it in the equipment room previously. I searched for it while holding on to that sliver of hope."
 <Page6>
-'Sliver of hope' is a bit of an exaggeration... I smile at that statement before noticing the slight increase in Hisui's breathing.
-She must have hurried to get it in the small time window while I was changing.
+'Sliver of hope' is a bit of an exaggeration... I smile at the turn of phrase before noticing that Hisui's breathing is a little elevated.
+She must have hurried to fetch it during the small time window while I was changing.
 <Page7>
-"Thank you. To be honest, it felt like I was going to jump from my heel with every step. You've been a great help."
+"Thank you. To be honest, I felt as if my foot was about to fall off with every step. This is a lifesaver."//This is pretty liberal, but I'm not sure of a good direct idiom for カカトから飛び上がりそうだった, and a literal TL just doesn't make sense
 "Of course. I am glad to be of service."
 <Page8>
-The weight of my left leg is transferred to my left hand on the walker.
-With this, I can walk without an issue. I thank Hisui for her thoughtfulness once again and move toward the lobby with her.
+I shift the weight of my body off of my left leg, and onto my left hand through the crutch.//左脚にかかる体重を、歩行器越しに左手で受け止める。
+With this, I can walk without issue. I thank Hisui once again for her thoughtfulness, and together we head toward the lobby.
 <Page9>
-―――Well, we've arrived.
+―――Well, we've arrived.//Thoughts on how to encode the ~てしまう? 到着してしまった。
```

Comment:
I think "well" already has the sort of exasperated "unfortunately" i get from this line